modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -95,13 +101,19 @@ pullSecret: '<pull_secret>' <6>
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
```

Comment:
In the step on [line 97](https://github.com/openshift/openshift-docs/pull/84551/files#diff-69efedd831fae26c26fdbda29b3ef4d1eef22ac76460b6f6cec52f4bd62ab949R97): If it was valid to add a step about creating an installation directory earlier, should this `openshift` directory be a subdirectory of that?